### DISCUSSION TITLE
What cicd are folks using with this to update their package on Chrome Store today?

### DISCUSSION THREAD

**Discussion by jacebenson** [crxjs/chrome-extension-tools#622] [General]
I just rewrote a hand-made extension using this and found it a delight, that is until I shipped the dev version (./dist) folder without doing a `npm run build`.

Now that I have that handled, I was wondering what github actions folks are using to automatically update their chrome store package or if folks are just having github zip it up with a build command?

**Comment by jacebenson** [comment]
https://jam.dev/blog/automating-chrome-extension-publishing/ Looks promising

**Reply by rileytaylor** [reply]
Haven't gotten to the upload part of our CI/CD process and we're currently just artifacting zips to manually upload. That [chrome-webstore-upload](https://github.com/fregante/chrome-webstore-upload) package which is used by that action is what I was planning on using

**Comment by JoeyDoey** [comment]
Haven't used it but the guys at @PlasmoHQ have their [Browser Platform Publisher](https://github.com/marketplace/actions/browser-platform-publisher) Github Action 👀
